lib/src/util.dart

```diff
@@ -110,3 +114,36 @@ dynamic transformDates(dynamic value) {
   }
   return value;
 }
+
+/// Compares two values, handles Lists and Maps
+/// by recursively calling itself on each entry.
+/// For Lists:
+///   - length has to be the same.
+///   - for each index, value in the other List at that index must be the same.
+/// For Maps:
+///   - keys must be the same.
+///   - for each key, value in the other Map at that key must be the same.
+/// For other types uses default implementation.
+bool deepEqual(dynamic v1, dynamic v2) {
```

Comment:
How is this different from https://api.flutter.dev/flutter/package-collection_collection/DeepCollectionEquality-class.html? Can you use this instead?

Reply:
Thanks, did not know about this part of ```collection``` package.
I suppose this can (and should) be used, will check on that today.

Reply:
This seems to work perfectly. I kept the utility method though, for convenience.

Reply:
Btw, you could keep it by writing something a tiny bit more concise:
```dart
final deepEqual = DeepCollectionEquality().equals;
```
Up to you.